src/theme/material/calendar.m.css

```diff
@@ -90,6 +91,11 @@
 	background-color: var(--mdc-theme-primary);
 }
 
+.date:not(.selectedDate):not(.inactiveDate):hover:before {
+	background-color: var(--mdc-theme-primary);
+	opacity: 0.04;
```

Comment:
can you beef this up to around `0.15`. On lower contrast screens this is currently invisible.